src/Persistence.Linnapps/ITransactionManager.cs

```diff
@@ -0,0 +1,7 @@
+﻿namespace Linn.Products.Persistence.Linnapps
+{
+    public interface ITransactionManager
+    {
```

Comment:
this is already in Linn.Common

Reply:
We already declared it separately in Discounting as well but I'll pull it in